### PR TITLE
Infolists: Add `KeyValueEntry` component

### PR DESCRIPTION
**Pull request by juliomotol**
## Description

This PR introduces `KeyValueEntry` which is similar to Form's `KeyValue`.

- [x] Visual changes (if any) are shown using screenshots/recordings of before and after.

image
image

## Code style

- [x] `composer cs` command has been run.

## Testing

- [x] Changes have been tested.

## Documentation

> Will write the docs when API is finalized.

- [ ] Documentation is up-to-date.

### CODE REVIEW

File: packages/infolists/resources/views/components/key-value-entry.blade.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<x-dynamic-component :component="$getEntryWrapperView()" :entry="$entry">
+    @php
+        $state = $getState();
+    @endphp
+
+    <div
+        {{
+            $attributes
+                ->merge($getExtraAttributes(), escape: false)
+                ->class(['fi-key-value w-full rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-white/5 dark:ring-white/10'])
```
Comment (zepfietje) [inline packages/infolists/resources/views/components/key-value-entry.blade.php]:
```suggestion
                ->class(['fi-in-key-value w-full rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-white/5 dark:ring-white/10'])
```

File: packages/infolists/resources/views/components/key-value-entry.blade.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                <tr>
+                    <th
+                        class="px-3 py-2 text-start text-sm font-medium text-gray-700 dark:text-gray-200"
+                        scope="col"
+                    >
+                        {{ $getKeyLabel() }}
+                    </th>
+
+                    <th
+                        class="px-3 py-2 text-start text-sm font-medium text-gray-700 dark:text-gray-200"
```
Comment (zepfietje) [inline packages/infolists/resources/views/components/key-value-entry.blade.php]: Please move `class` to be the last attribute (in all places in this file).

**Comment by danharrin** [comment]
Thanks!

File: packages/infolists/resources/views/components/key-value-entry.blade.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<x-dynamic-component :component="$getEntryWrapperView()" :entry="$entry">
+    <div
+        {{
+            $attributes
+                ->merge($getExtraAttributes(), escape: false)
+                ->class(['fi-in-key-value w-full rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-white/5 dark:ring-white/10'])
```
Comment (zepfietje) [inline packages/infolists/resources/views/components/key-value-entry.blade.php:6]: @juliomotol @danharrin rounding is wrong here.

Comment (zepfietje) [reply packages/infolists/resources/views/components/key-value-entry.blade.php:6]: Fixed it on feature-staging.

**Pull request merged**